File: Java/opentraceability/src/main/java/opentraceability/utility/CachedXmlSchema.java
package opentraceability.utility;

import javax.xml.validation.Schema;
import java.time.OffsetDateTime;
import java.time.ZoneOffset;
import java.time.ZonedDateTime;

public class CachedXmlSchema {
    public String url = null;
    public Schema schemaSet = null;
    public OffsetDateTime lastUpdated = OffsetDateTime.now(ZoneOffset.UTC);

    public CachedXmlSchema() {}

    public CachedXmlSchema(String url, Schema schemaSet) {
        this.url = url;
        this.schemaSet = schemaSet;
    }
}

File: Java/OpenTraceability/src/main/opentraceability/utility/cachedxmlschema.kt
package utility

import javax.xml.validation.Schema
import java.time.ZonedDateTime
import java.time.ZoneOffset

class CachedXmlSchema {
    var url: String? = null
    var schemaSet: Schema? = null
    val lastUpdated: ZonedDateTime = ZonedDateTime.now(ZoneOffset.UTC)

}
